README.md

```diff
@@ -1806,6 +1806,7 @@ HTML Themes and UI Kits to create your applications faster!
   * [AdminMart](https://adminmart.com/) — High Quality Free and Premium Admin Dashboard and Website Templates created with Angular, Bootstrap, React, VueJs, NextJs and NuxtJs!
   * [Glob tester](https://globster.xyz/) — A website that allows you to design and test glob patterns. It also provides resources to learn glob patterns.
   * [OpenUtils](https://openutils.org/) - There are various free tools available for developers, such as HTML/CSS/JavaScript formatters, minifiers, converter, encoder/decoder and many others.
+  * [MeteorOps](https://meteorops.com/) - Free DevOps consultations and project estimations.
```

Comment:
Indeed started as a consultancy, but in the process of it turning into a marketplace, it released free offerings for consultations and project estimations.

Opened this PR as multiple developers used this offering before, and it enabled them to push forward various DevOps efforts within their companies.

Reply:
I'm aware that this might not be a trivial free offering to add to the list, but I figured that if it's a free offering that helps developers, and is in the process of gradually turning into a productized marketplace, it might actually fit in this exact list.

Reply:
I dont think its suitable to the list. Not to say its not useful, it's just not a SaaS.

Reply:
I respect that, thank you for considering it, and will check again in the future when it's more productized as a SaaS.